src/pages/Patient/components/PatientModal.tsx

```diff
@@ -163,22 +167,30 @@ export const PatientModal: FC<PatientModalProps> = ({
 
     const isFormValid = validateForm();
```

Comment:
Try not  to  write  nested  if

Reply:
if(isFormValid) {
(selectedPatient?updatePatient:createPatient)( { ...patientForm, image: patientImage, _id: selectedPatient._id,     })
}